x-pack/test/functional/apps/monitoring/elasticsearch/shards.js

```diff
@@ -33,6 +33,8 @@ export default function ({ getService, getPageObjects }) {
         // start on cluster overview
         await PageObjects.monitoring.clickBreadcrumb('breadcrumbClusters');
 
+        await PageObjects.header.waitUntilLoadingHasFinished();
```

Comment:
Is this related to these changes at all or just a general flakiness problem?

Reply:
A general flakiness problem that the slight increase in latency these changes introduced have made more flaky.